packages/vc-test-suite/helpers.ts

```diff
@@ -35,3 +35,27 @@ export function setupMatrix(implementors: string[], columnLabel: string) {
   const images = [];
   this.images = images;
 }
+
+/**
+ * Builds a URL with additional query parameters.
+ * @param baseUrl - The base URL.
+ * @param additionalParams - Additional query parameters as key-value pairs.
+ * @returns The built URL as a string.
+ */
+export const buildUrl = (baseUrl: string, additionalParams: Record<string, any>) => {
+  const url = new URL(baseUrl);
+  Object.entries(additionalParams).forEach(([key, value]) => {
+    url.searchParams.append(key, value);
+  });
+  return url.toString();
+};
+
+/**
+ * Builds a payload by merging a base payload with additional payload.
+ * @param basePayload - The base payload object.
+ * @param additionalPayload - The additional payload object.
+ * @returns The merged payload object.
+ */
+export const buildPayload = (basePayload: Record<string, any>, additionalPayload: Record<string, any>) => {
+  return { ...additionalPayload, ...basePayload };
```

Comment:
Hello, should we check `basePayload`  and `additionalPayload` are objects and they are not null?